package.json

```diff
@@ -38,7 +38,7 @@
     "lint-source": "eslint --color --ext .js,.jsx src",
     "lint-demo": "eslint --color --ext .js,.jsx demo",
     "lint-docs": "eslint --color --ext .js,.jsx docs",
-    "lint-test": "eslint --color --ext .js,.jsx test",
+    "lint-test": "eslint --color --ext .js,.jsx --rule 'mocha/no-exclusive-tests:2' test",
```

Comment:
We shouldn't have a command line rule here. it should be in `config/eslint/.eslintrc-test`

**But**, it appears on all these lints we're _not_ using our configs???